x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -82,6 +101,38 @@ export class TelemetryEventsSender {
     }
   }
 
+  public async fetchDiagnosticAlerts() {
+    const query = {
+      expand_wildcards: 'open,hidden',
+      index: 'logs-endpoint.diagnostic.collection-default*',
+      ignore_unavailable: true,
+      size: this.maxQueueSize,
+      body: {
+        query: {
+          range: {
+            'event.ingested': {
+              gte: 'now-5m',
```

Comment:
It's possible that you will miss some events this way... the task will try to run every 5 minutes, but it's not a guarantee. It may be best to perform a `max` aggregation to get the timestamp of the last event that matches, and then pass that through as `state` to the next task run to use as a lower bound...

Reply:
But if missing an event occasionally is acceptable, then disregard. :)

Reply:
Thanks @madirey! We discussed and agree. See: 5638da9

Reply:
Looks great!